helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBucketDataAccessor.java

```diff
@@ -112,6 +118,7 @@ public void testManyWritesWithVersionCounts() throws IOException {
 
     // Clean up
     bucketDataAccessor.compressedBucketDelete(path);
+    bucketDataAccessor.disconnect();
```

Comment:
np, add a check for verifying ZkClient closed after disconnect?

Reply:
zkclient.close is a synchronized call. Also we don't expose the zk clients at the accessor level so that is difficult to do and probably not necessary.